src/EmployeeBook.java

```diff
@@ -0,0 +1,232 @@
+import java.util.*;
+import java.util.ArrayList;
+
+// Хранилище записей о сотрудниках
+public class EmployeeBook {
+
+    // Записи о сотрудниках
+    private Employee[] arrayEmployees;
+
+
+
+    // Конструктор
+    public EmployeeBook() {
+        this.arrayEmployees = new Employee[10];
+        fill();
+    }
+
+
+
+    // Заполнить книгу
+    public void fill()
+    {
+        arrayEmployees[0] = new Employee("Иванов Иван Иванович", 4, 50000);
+        arrayEmployees[1] = new Employee("Иванов Иван Иванович", 3, 50000);
+        arrayEmployees[2] = new Employee("Павлова Екатерина Ивановна", 2, 70000);
+        arrayEmployees[3] = new Employee("Антонова Марина Сергеевна", 1, 90000);
+        arrayEmployees[4] = new Employee("Сергеев Семён Петрович", 5, 90000);
+       /* arrayEmployees[5] = new Employee("Иванов Александр Иванович", 1, 40000);
+        arrayEmployees[6] = new Employee("Петров Семён Александрович", 3, 50000);
+        arrayEmployees[7] = new Employee("Семёнова Елена Ивановна", 2, 60000);
+        arrayEmployees[8] = new Employee("Антонова Марина Сергеевна", 4, 80000);
+        arrayEmployees[9] = new Employee("Сергеев Семён Алексеевич", 1, 50000);*/
+    }
+
+    // Добавить нового сотрудника
+    public String addEmployee(Employee employeeToAdd) {
+        boolean isEmployeeExist = (Arrays.stream(arrayEmployees).filter(emp -> emp.getId() == employeeToAdd.getId())) == null;
+        if(isEmployeeExist)
+        {
+            return ("Сотрудник был добавлен ранее.");
+        }
+        else {
+            int cellCounter = 0;
+            for (Employee e : arrayEmployees) {
```

Comment:
Тебе бы было проще обычный цикл for использовать

```suggestion
            for (int cellCounter=0; cellCounter < arrayEmployees.length; cellCounter++) {
```